frontend/public/components/app.jsx

```diff
@@ -248,10 +248,10 @@ window.onunhandledrejection = function (e) {
 };
 
 if ('serviceWorker' in navigator) {
-  if (window.location.search.indexOf('loadTest') > -1) {
+  if (window.SERVER_FLAGS.loadTestFactor > 1) {
     import('file-loader?name=load-test.sw.js!../load-test.sw.js')
       .then(() => navigator.serviceWorker.register('/load-test.sw.js'))
-      .then(() => navigator.serviceWorker.controller.postMessage({topic: 'setFactor', value: Number(new URLSearchParams(window.location.search).get('loadTest'))}));
+      .then(() => navigator.serviceWorker.controller.postMessage({topic: 'setFactor', value: window.SERVER_FLAGS.loadTestFactor}));
```

Comment:
Can you add a check here for <code>controller</code>? 
![screen shot 2018-05-23 at 11 57 59 am](https://user-images.githubusercontent.com/523368/40445327-ab6c2a22-5e80-11e8-9fac-f87cd2b2c173.png)

Reply:
Good call, fixed it.